### PR TITLE
Ping: Use continuous start

### LINKED ISSUE #142: Add support for free-running mode
Ping hardware has a command to enable continuous (non-polled) data stream. Try it out!

### PR DESCRIPTION
**Pull request by patrickelectric**
Use only _all header

Fix #142

Signed-off-by: Patrick José Pereira <patrickelectric@gmail.com>

### CODE REVIEW

File: src/sensor/ping.cpp [hunk 1]
```diff
@@ -405,30 +384,22 @@ void Ping::request(int id)
...
-        return 0;
-    }
-    return 1000.0f / _requestTimer.interval();
+    return _pollFrequency;
 }
 
 void Ping::setPollFrequency(float pollFrequency)
 {
-    if (pollFrequency <= 0 || pollFrequency > 30) {
+    if (pollFrequency <= 0 || pollFrequency > 50) {
```
Comment (jaxxzer) [inline src/sensor/ping.cpp]: Did you want to update the qml too?

Comment (patrickelectric) [reply src/sensor/ping.cpp]: Done via pollMaxFrequency.

**Comment by patrickelectric** [comment]
Correction: The problem was a variable created probably from a rebase.

File: src/sensor/ping.h [hunk 2]
```diff
@@ -445,6 +474,7 @@ class Ping : public Sensor
 
     bool _mode_auto = 0;
     uint16_t _ping_rate = 0;
+    float _pollMaxFrequency = 50;
```
Comment (jaxxzer) [inline src/sensor/ping.h]: static const?

File: qml/MainMenu.qml [hunk 3]
```diff
@@ -98,7 +98,7 @@ Item {
                         id: pingHzSlider
                         from: 0
                         stepSize: 1
-                        to: 30
+                        to: ping.pollMaxFrequency
                         value: ping.pollFrequency;
```
Comment (jaxxzer) [inline qml/MainMenu.qml]: This should be `1/ping.msec_per_ping`

File: src/sensor/ping.cpp [hunk 4]
```diff
@@ -405,30 +384,22 @@ void Ping::request(int id)
 
 float Ping::pollFrequency()
```
Comment (jaxxzer) [inline src/sensor/ping.cpp]: We can remove this member. We are no longer polling!

**Pull request merged**